src/docfx/build/xref/InternalXrefSpec.cs

```diff
@@ -12,6 +12,8 @@ internal class InternalXrefSpec : IXrefSpec
     {
         public SourceInfo<string> Uid { get; }
 
+        public string? SchemaType { get; private set; }
```

Comment:
Add `string? SchemaType { get; }` to `IXrefSpec`?

Reply:
OK.